.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+name: main branch
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'api/**'
```

I think we also need to add
```suggestion
      - '/*.go'
```